src/store/store.ts

```diff
@@ -591,14 +591,16 @@ export const useStore = defineStore('store', {
       // *** TODO: add checks for review page components
       const isFolioNumberValid = !this.isPremiumAccount || this.getFolioNumberValid
       const isCourtOrderValid = this.isRoleStaff ? this.getCourtOrderStep.valid : true
+      const isCourtApprovalValid = this.getAmalgamationCourtApprovalValid
```

Comment:
A temporary variable makes sense for lines 592 and 593 since they do some computation and it looks cleaner to not have that on lines 601 and 602.

In your case, I recommend just putting the getter on line 603.

Reply:
Looks good.